modules/node/init.zsh

```diff
@@ -6,13 +6,43 @@
 #   Zeh Rizzatti <zehrizzatti@gmail.com>
 #
 
+lazy_load_nvm() {
+  NVM_DIR=$1
+  # Skip adding binaries if there is no node version installed yet
+  if [ -d $NVM_DIR/versions/node ]; then
+    NODE_GLOBALS=(`find $NVM_DIR/versions/node -maxdepth 3 \( -type l -o -type f \) -wholename '*/bin/*' | xargs -n1 basename | sort | uniq`)
```

Comment:
Does this work on OSX as well? I seem to remember the find command being fairly limited.